docs/qupath.rst

```diff
@@ -57,11 +67,19 @@ Step-by-step
 
    |image1|
 
+#. Click ``Import``.
+
+#. The image thumbnail will appear in the left-hand pane list of the QuPath Project. Click on that thumbnail to open the image in full viewer in central pane of QuPath.
```

Comment:
Feels like it needs more "the". The full viewer's actually in the central pane?

Reply:
> The full viewer's actually in the central pane?

Thank you for that, actually, QuPath does not have a right-hand pane, it just have a smaller left-hand pane and a much bigger right-hand pane.
I will remove the central pane wording.

Reply:
Fixed in https://github.com/ome/omero-guide-qupath/pull/4/commits/9d0bf5f1005e50fc539d3b04615e942b3cb0f302 - is that any better ?